org.kde.kst.yml

```diff
@@ -0,0 +1,112 @@
+app-id: org.kde.kst
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: kst2
+rename-desktop-file: kst2.desktop
+rename-icon: kst
+finish-args:
+  - '--share=network'
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--filesystem=home'
```

Comment:
This doesn't seem needed – it appears to use the portal correctly on my GNOME system.

Reply:
Not quite sure about this one, could it break other systems if removed?

Reply:
Whats the diffrence in that regard between your GNOME system or someone elses plasma/xfce.. etc?

Reply:
At least major distributions pull in portals implementation when installing flatpak so it should work everywhere.